### PR TITLE
Fix Linux dark theme

### PR DESCRIPTION
**Pull request by jsanko**
IB-6991
Signed-off-by: Jelena Sanko <jelena@raulwalter.com>

### CODE REVIEW

File: client/dialogs/SettingsDialog.ui [hunk 1]
```diff
@@ -1108,21 +1108,21 @@ QScrollBar::sub-line:vertical {
...
-&lt;p align=&quot;center&quot;&gt;&lt;b&gt;JSON for Modern C++&lt;/b&gt;&lt;br /&gt;MIT License&lt;br /&gt;&lt;a href=&quot;https://github.com/nlohmann/json/blob/develop/LICENSE.MIT&quot; style=&quot; text-decoration: underline; color:#006eb5;&quot;&gt;https://github.com/nlohmann/json/blob/develop/LICENSE.MIT&lt;/a&gt; &lt;/p&gt;
-&lt;p align=&quot;center&quot;&gt;&lt;b&gt;Roboto: Google’s signature family of fonts&lt;/b&gt;&lt;br /&gt;Apache Software License, Version 2.0&lt;br /&gt;&lt;a href=&quot;https://github.com/google/roboto/blob/master/LICENSE&quot; style=&quot; text-decoration: underline; color:#006eb5;&quot;&gt;https://github.com/google/roboto/blob/master/LICENSE&lt;/a&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+p, li { white-space: pre-wrap; }
```
Comment (metsma) [inline client/dialogs/SettingsDialog.ui:1111]: Please revert this change

**Review by metsma — CHANGES_REQUESTED** [review]

**Review by metsma — APPROVED** [review]

File: client/dialogs/SettingsDialog.ui (hunk 1 above)
Comment (jsanko) [reply client/dialogs/SettingsDialog.ui]: reverted

**Review by metsma — APPROVED** [review]

File: client/dialogs/MobileDialog.ui [hunk 2]
```diff
@@ -11,7 +11,11 @@
...
   </property>
   <property name="styleSheet">
-   <string notr="true">QDialog {
+   <string notr="true">QWidget {
+background-color: #FFFFFF;
+color: #000000;
+}
+QDialog {
 border-radius: 2px;
 background-color: #FFFFFF;
```
Comment (metsma) [inline client/dialogs/MobileDialog.ui:20]: You can remove this

File: client/dialogs/RoleAddressDialog.cpp [hunk 3]
```diff
@@ -83,6 +84,7 @@ RoleAddressDialog::RoleAddressDialog(QWidget *parent)
 			d->s.setValue(line->objectName(), QString()); // Uses on Windows MULTI_STRING registry
 			SettingsDialog::setValueEx(line->objectName(), list, QStringList());
 		});
+		completer->popup()->setStyleSheet("background-color:#FFFFFF; color: #000000;");
```
Comment (metsma) [inline client/dialogs/RoleAddressDialog.cpp:87]: Can you move this CSS to UI file

File: client/dialogs/SettingsDialog.cpp [hunk 4]
```diff
@@ -110,7 +110,13 @@ SettingsDialog::SettingsDialog(QWidget *parent)
 	ui->lblAccessCert->setFont(regularFont);
 	ui->txtAccessCert->setFont(regularFont);
 	ui->btInstallManually->setFont(condensed12);
+#ifdef Q_OS_LINUX
+	ui->btInstallManually->setStyleSheet("background-color: #d3d3d3");
+#endif
 	ui->btShowCertificate->setFont(condensed12);
+#ifdef Q_OS_LINUX
+	ui->btShowCertificate->setStyleSheet("background-color: #d3d3d3");
+#endif
```
Comment (metsma) [inline client/dialogs/SettingsDialog.cpp:119]: Use one ifdef

File: client/dialogs/SmartIDDialog.ui [hunk 5]
```diff
@@ -137,6 +138,9 @@ left: 1px;
        <height>34</height>
       </size>
      </property>
+     <property name="styleSheet">
+      <string notr="true"/>
+     </property>
```
Comment (metsma) [inline client/dialogs/SmartIDDialog.ui:143]: revert

**Review by metsma — CHANGES_REQUESTED** [review]

File: client/dialogs/SmartIDDialog.ui (hunk 5 above)
Comment (jsanko) [reply client/dialogs/SmartIDDialog.ui]: reverted

File: client/dialogs/RoleAddressDialog.cpp (hunk 3 above)
Comment (jsanko) [reply client/dialogs/RoleAddressDialog.cpp]: moved

File: client/dialogs/MobileDialog.ui (hunk 2 above)
Comment (jsanko) [reply client/dialogs/MobileDialog.ui]: removed

**Review by metsma — APPROVED** [review]

**Review by metsma — APPROVED** [review]

**Pull request merged**